src/Components/AlbumCard/AlbumCard.css

```diff
@@ -11,37 +11,75 @@
   background: radial-gradient(ellipse at 13% 0, rgba( 205, 214, 221, 0.25 ) 25%, transparent 45%, rgba( 205, 214, 221, 0.25 ));
   backdrop-filter: blur( 1px );
   -webkit-backdrop-filter: blur( 1px );
+  transition: transform .6s;
 }
  
-.cover {
+.card_card:hover {
+  transform: scale(1.1);
+}
+
+
+.card_details-link-container {
+  color: var(--text-color);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-radius: 7px;
+  padding-bottom: 1vh;
+  /* text-decoration: none; */
+}
+
+.card_details-link-container:hover {
+  cursor: pointer;
+  width: 100%;
+}
+
+/* 
+.title:hover, .artist:hover, .date:hover {
+  text-decoration: underline;
+} */
```

Comment:
Since we are setting a hover state to the entire card now I think we can remove this